.cursor/rules/stories-format.mdc

```diff
@@ -0,0 +1,465 @@
+    args: {},
+};
+```
+
+### States stories
+
+Stories demonstrating different states (if applicable):
+
+```typescript
+/**
+ * Description of this particular state.
+ */
+export const StateName: Story = {
+    render: (args) => html`
+        ${template({ ...args /* state example */ })}
+    `,
+    tags: ['states'],
+    args: {},
+};
+```
+
+### Options stories
+
+Stories demonstrating variants, sizes, styles:
+
+```typescript
+/**
+ * Description of this option/variant.
+ */
+export const OptionName: Story = {
+    render: () => html`
+        ${/* examples */}
+    `,
+    tags: ['options'],
+};
+```
+
+### Accessibility story
+
+Required accessibility demonstration:
+
+```typescript
+/**
+ * ### Features
+ *
+ * The `<sp-component-name>` element implements several accessibility features:
+ *
+ * 1. **Feature name**: Description
+ * 2. **Feature name**: Description
+ *
+ * ### Best Practices
+ *
+ * - Best practice one
+ * - Best practice two
+ */
+export const Accessibility: Story = {
+    render: () => html`
+        ${/* accessibility examples */}
+    `,
+    tags: ['a11y'],
+};
+```
+
+## Tag usage requirements
+
+### Required tags
+
+- `'autodocs'` - Only on Playground story
+- `'dev'` - Only on Playground story
+- `'anatomy'` - For anatomy stories
+- `'states'` - For state demonstration stories (if component has states)
+- `'options'` - For variant, size, and style stories
+- `'a11y'` - For accessibility demonstration story
+
+### Exclusion tags
+
+- `'!test'` - Exclude from test runs (e.g., complex static color stories)
+- `'!dev'` - Exclude from dev Storybook
+
+### Migration tag
+
+- `'migrated'` - On the meta object to indicate refined format
+
+## JSDoc requirements
+
+### Above each story export
+
+Every story export must have a JSDoc comment explaining:
+
+- What it demonstrates
+- Any important context or usage notes
+- Best practices if relevant
+
+**Exception**: Do NOT add JSDoc comments above the Playground story.
+
+Use markdown formatting within JSDoc:
+
+- `**Bold**` for emphasis
+- Bullet lists for multiple points
+- Code formatting with backticks
+
+Example:
+
+```typescript
+/**
+ * Semantic variants allow you to render the badge with a descriptive name that maps to a design-system-aligned color.
+ *
+ * Use these variants for the following statuses:
+ * - **Positive**: approved, complete, success, new, purchased, licensed
+ * - **Informative**: active, in use, live, published
+ * - **Negative**: error, alert, rejected, failed
+ */
+export const SemanticVariants: Story = {
+    // ...
+};
+```
+
+## Static color pattern
+
+For components with `static-color` attribute, implement the three-story pattern:
+
+```typescript
+// Individual color stories without JSDoc
+export const StaticWhite: Story = {
+    args: {
+        'static-color': 'white',
+        // other args
+    },
+};
+
+export const StaticBlack: Story = {
+    args: {
+        'static-color': 'black',
+        // other args
+    },
+};
```

Comment:
It owuld be good to have the static colors in a single story so it's easier to visually compare them.